lib/src/main/java/space/taran/arklib/Lib.kt

```diff
@@ -1,18 +1,22 @@
-package space.taran.arklib;
+package space.taran.arklib
 
 import android.graphics.Bitmap
+import java.nio.file.Path
 
-public data class LinkData(
+data class LinkData(
    val title: String,
    val desc: String,
    val url: String,
    val imageUrl: String){
    companion object {
      @JvmStatic
-     fun create(title: String, desc: String, url: String, imageUrl: String) : LinkData? = LinkData(title, desc, url, imageUrl)
+     fun create(title: String, desc: String, url: String, imageUrl: String) : LinkData = LinkData(title, desc, url, imageUrl)
   }
 }
 
+typealias ResourceId = Long
```

Comment:
We should think what is needed to migrate from
```
ResourceId = Long
```
to
```
ResourceId = Pair<Long, Long>
```
where second Long value is filesize.

That would allow us to remove this typealias and just use `ResourceId` from `arklib`.

Reply:
Created separate issue for this:
https://github.com/ARK-Builders/arklib-android/issues/20